docs/Microservices/overview.md

```diff
@@ -84,6 +84,27 @@ For more information about power states supported by the REST APIs, see [Intel®
 
 Intel® AMT enables remote management of a device, even when the OS isn't running, through KVM over IP support. No additional equipment is needed for this feature.  With KVM control, IT administrators can access and update PCs and devices as if they were onsite. It eliminates the need for remote KVM switches and other hardware. 
 
+## Passwords
+
+There are four levels of passwords: 
+
+1. **Intel® Manageability Engine BIOS Extensions (MEBX) Password : MEBX Password - ** Use this password to secure the local Intel® MEBX menu. This password is only used when physically accessing the managed device during system boot. Access the menu with Ctrl-P on most devices.
+2. **Sample Web UI Password: MPS_WEB_ADMIN_PASSWORD - ** Use this password on the development system when logging into the Sample Web UI. The Sample Web UI Password uses this default MPS user authentication credential when it triggers MPS to issue a JWT (JSON Web Token). In most production environments, this default credential is replaced by a more rigorous authentication protocol. Examples include OAuth2, OpenID, or an authentication service that can issue an Auth Token to be validated by the API gateway. 
```

Comment:
minor item, I would remove "on the development system" from the first sentence.  You can log into the sample web ui from anywhere that has network access to the container running the sample web ui.  For example, when it is running in Azure.

Reply:
Update to read "**Sample Web UI Password: MPS_WEB_ADMIN_PASSWORD - ** User this password when logging into the Sample Web UI."